examples/storm-starter/src/jvm/org/apache/storm/starter/WordCountTopology.java

```diff
@@ -90,7 +90,9 @@ public static void main(String[] args) throws Exception {
     if (args != null && args.length > 0) {
       conf.setNumWorkers(3);
 
-      StormSubmitter.submitTopologyWithProgressBar(args[0], conf, builder.createTopology());
+      for (String name: args) {
```

Comment:
I'm not sure I understand the for loop here. Isn't this change just submitting the same topology multiple times with different names? Is this an artifact from manual testing?